scripts/repair_corrupted_waves.py

```
@@ -0,0 +1,335 @@
+    ids = ", ".join(f'ObjectId("{doc_id}")' for doc_id in repair["drop_doc_ids"])
+    js_lines.append(
+        "db.deltas.deleteMany({_id: {$in: [" + ids + "]}});"
+    )
+    js_lines.append(
+        "db.snapshots.deleteMany({waveId: "
+        + json.dumps(repair["waveid"])
+        + ", waveletId: "
+        + json.dumps(repair["waveletid"])
+        + "});"
+    )
+  proc = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)